NextcloudTalk/NCChatViewController.m

```diff
@@ -1273,7 +1273,7 @@ - (void)presentAttachmentsOptions
         [optionsActionSheet addAction:cameraAction];
     }
     [optionsActionSheet addAction:photoLibraryAction];
-    if ([[NCDatabaseManager sharedInstance] serverHasTalkCapability:kCapabilityTalkPolls]) {
+    if ([[NCDatabaseManager sharedInstance] serverHasTalkCapability:kCapabilityTalkPolls] && _room.type != kNCRoomTypeOneToOne) {
```

Comment:
Are you checking chat permissions? (similar as for other attachments)

Reply:
if the user doesn't have chat permission the input field (and the attachments button) is hidden.
So the user won't be able to select any attachment option.